docs/config.md

```diff
@@ -514,6 +514,28 @@ conflict is considered fully resolved when there are no conflict markers left.
 
 ## Git settings
 
+### Default remotes for `jj git fetch` and `jj git push`
+
+By default, the default remote used for `jj git fetch` and `jj git push` is the
```

Comment:
Nit: Sentence can be improved, "By default the remote used for ... is the remote called origin just like Git."